### DISCUSSION TITLE
mouse wheel not working after update

### DISCUSSION THREAD

**Discussion by tomysurya** [YaLTeR/niri#788] [Q&A] [answered]
hello guys, i need help after update to niri 0.1.9 (v0.1-9-95-g8867a4f) my mouse wheel didn't work, before update it was working just fine, any help will be appreciated, thank you

**Comment by YaLTeR** [comment]
What was the commit that you updated *from*? Could you attach the output of `sudo libinput list-devices`?

**Comment by tomysurya** [comment]
Device:           Logitech USB Optical Mouse
Kernel:           /dev/input/event2
Group:            4
Seat:             seat0, default
Capabilities:     pointer
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      disabled
Nat.scrolling:    disabled
Middle emulation: disabled
Calibration:      n/a
Scroll methods:   button
Click methods:    none
Disable-w-typing: n/a
Disable-w-trackpointing: n/a
Accel profiles:   flat *adaptive custom
Rotation:         0.0

**Reply by YaLTeR** [reply]
Could you record scrolling in `sudo libinput debug-events` and post the events here?

**Comment by tomysurya** [comment]
-event1   DEVICE_ADDED            Power Button                      seat0 default group1  cap:k
-event4   DEVICE_ADDED            Video Bus                         seat0 default group2  cap:k
-event0   DEVICE_ADDED            Power Button                      seat0 default group3  cap:k
-event2   DEVICE_ADDED            Logitech USB Optical Mouse        seat0 default group4  cap:p left scroll-nat scroll-button
-event6   DEVICE_ADDED            Eee PC WMI hotkeys                seat0 default group5  cap:k
-event3   DEVICE_ADDED            AT Translated Set 2 keyboard      seat0 default group6  cap:k
-event2   POINTER_SCROLL_WHEEL    +4294967.295s	vert 15.00/120.0* horiz 0.00/0.0 (wheel)
 event2   POINTER_SCROLL_WHEEL    +0.655s	vert 15.00/120.0* horiz 0.00/0.0 (wheel)
 event2   POINTER_SCROLL_WHEEL    +1.239s	vert 15.00/120.0* horiz 0.00/0.0 (wheel)
 event2   POINTER_SCROLL_WHEEL    +7.199s	vert -15.00/-120.0* horiz 0.00/0.0 (wheel)
 event2   POINTER_SCROLL_WHEEL    +7.815s	vert -15.00/-120.0* horiz 0.00/0.0 (wheel)
 event2   POINTER_SCROLL_WHEEL    +8.479s	vert -15.00/-120.0* horiz 0.00/0.0 (wheel)
 event2   POINTER_MOTION          +9.295s	  0.00/ -0.30 ( +0.00/ -1.00)
 event2   POINTER_MOTION          +9.391s	  0.38/ -0.38 ( +1.00/ -1.00)
-event3   KEYBOARD_KEY            +14.265s	KEY_RIGHTCTRL (97) pressed
 event3   KEYBOARD_KEY            +14.533s	*** (-1) pressed

thats a little wheel scroll up and down, but it's not working

**Reply by YaLTeR** [reply, marked as answer]
That looks normal. Which niri version did it work on? Can you post your niri config?

**Comment by tomysurya** [comment]
oh my! forgot to insert mouse  on the niri config input, and now it works! thank you

**Reply by YaLTeR** [reply]
Huuuh? That's not normal, it shouldn't break this way. Could you please also post `sudo libinput list-devices` with the working scroll wheel?

**Reply by YaLTeR** [reply]
Actually, no need, I just reproduced it.

**Reply by YaLTeR** [reply]
Fixed in 1951d2a9f262196a706f2645efb18dac3c4d6839.
